packages/web_benchmarks/pubspec.yaml

```diff
@@ -2,10 +2,10 @@ name: web_benchmarks
 description: A benchmark harness for performance-testing Flutter apps in Chrome.
 repository: https://github.com/flutter/packages/tree/main/packages/web_benchmarks
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+web_benchmarks%22
-version: 0.0.7
+version: 0.0.8
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: '>=2.12.0 <3.0.0'
   flutter: ">=2.10.0"
 
 # Using +2 upper limit on some packages to allow null-safe versions
```

Comment:
Is `meta` still actually used?

Reply:
currently being used for sealed annotations